### ISSUE TITLE
{Freeway-Bug}[Adaptive Card - Expense Report.Json] - Luminosity ratio of text Reject is '3.098:1' in collapsed and '1.263:1' in expanded states respectively with respect to the background, which is less than 4.5:1

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#47]
**Test Environment:**
Device: iPhone 11
iOS Version: 16.7.1
App Version: 1.0 (2.4.0-beta.23.10.15.1)
Screen Reader: Voice Over
Tool: Accessibility Insight For Windows

**Repro Steps:**
1. Launch Adaptive Cards iOS application.
2. Home screen will appear.
3. Navigate to "V1.5" control and select it.
4. Navigate on "V1.5" screen and select "Scenarios" button.
5. 'Scenario' screen will appear.
6. Navigate to "Expense Report.json" control and select it.
7. Check the CCA of Reject text with respect to background and verify the issue.

**Actual Result:**
Luminosity ratio of text Reject is '3.098:1' in collapsed and '1.263:1' in expanded states respectively with respect to the background, which is less than 4.5:1

**Expected Result:**
Luminosity ratio of text Reject with respect to the background, should be greater or equal to 4.5:1 in any state.

**User Experience:**
Low vision users will be impacted if without keyboard focus luminosity contrast ratio of Reject text with respect to background is less than as per expected.

MAS Reference:
https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum

"Have feedback to share on Bugs ? Please help fill [Trusted Tester Bug Feedback (office.com)](https://nam06.safelinks.protection.outlook.com/?url=https://forms.office.com/pages/responsepage.aspx?id%3Dv4j5cvGGr0GRqy180BHbR5dUqGQgJ9lItv-jqY8F09lURVdYTENIRkY2UzlCR0NKV1E2VUE5OVpDVy4u&data=04%7c01%7cv-apoorva%40microsoft.com%7c551850210d444be9b28308d9691e3dde%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c637656400998658942%7cUnknown%7cTWFpbGZsb3d8eyJWIjoiMC4wLjAwMDAiLCJQIjoiV2luMzIiLCJBTiI6Ik1haWwiLCJXVCI6Mn0%3D%7c1000&sdata=l6w0SvVcMSvyGnJ6%2B6uSSgd3braF2dZ7/AdBrqqzIR8%3D&reserved=0) “

![Color Contrasrt for Reject text is 3 098 1 which is less than 4 5 1](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/e6135a52-3f6c-439a-8ec4-5786ff199ac5)

![CCA for Reject button in expand state is 1 263 1, which is less than 4 5 1](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/aac0da97-8b59-41f9-b920-edf847d74049)

**Comment by vagpt** [comment]
Hi @GabrielMedAlv, @ardlank,

Could you please provide us with an updated build to test this issue? We have checked and recently we have the same version on the app center on which we have performed the Test Pass i.e. **1.0 (2.4.0-beta.23.10.15.1)**.

CC: @Raisul123

**Comment by ardlank** [comment]
@jwoo-msft Do you know how we can supply a new build for the tester?

**Comment by jwoo-msft** [comment]
you will have to publish your apps to AppCenter.

**Comment by vagpt** [comment]
Hi @ardlank,

Any update regarding the updated build?

**Comment by ardlank** [comment]
Have a meeting this week to enable these build for you

**Comment by vagpt** [comment]
This issue is now working fine hence closing this issue.

![image](https://github.com/user-attachments/assets/64fa3bef-bed8-43c3-8862-44f8d779f12f)


![image](https://github.com/user-attachments/assets/1ec7f166-0cd9-42c7-a7e5-de8cfbad39f3)
